specification/metadata/schemaid.md

```diff
@@ -0,0 +1,17 @@
+# Schema ID
+
+Unique identifier for the schema.
```

Comment:
Too vague

Reply:
Does the schema belong to the provider or to the data generator?

I would go the simple path where we don't split the metadata between provider and data generator: there would be only data generator metadata, and then it would be the data generator schema.

Reply:
Is this covered by providing a version of the schema? I was thinking of that in the metadata section.

Reply:
We need to decide between using SchemaID or DataGeneratorVersion as the identifier. I don't think we should have both and Im leaning towards SchemaID with more of the text pulled in from DataGeneratorVersion.  cc:// @rileyjenk

Reply:
We might need both Data Generator Versions and Schema ID: there can be different Data Generator Versions for the same Schema ID, for example if there are bug fixes in the Data Generator.

From practitioner perspective:
- The Schema ID is important for data ingestion to manage column changes.
- The Data Generator Version is important to spot potential data issues that can be linked to different Data Generator Versions.

Reply:
If one was to choose between Schema ID and Data Generator Version, I would vote for the Schema ID, as this is directly impacting the data ingestion. The Data Generator Version is more software management and can be managed outside FOCUS metadata.

Reply:
Opening related PR to discuss this further